### ISSUE TITLE
Displaysettings class name

### ISSUE THREAD

**Issue by tischi** [BIOP/bigdataviewer-image-loaders#6]
@NicoKiaru Any chance you could rename `Displaysettings` to `DisplaySettings`?
Or is there a reason for the current name?

**Comment by NicoKiaru** [comment]
Dude! I think that changed from [DisplaySettings to Displaysettings was explicitely asked by @constantinpape](https://github.com/bigdataviewer/bigdataviewer-playground/issues/111#issuecomment-617111394) to be compatible with pybdv (changed in commit https://github.com/bigdataviewer/bigdataviewer-playground/commit/c2558339734c29feb3841328537d37120fe56dea).

Changing the capitalization will break too many things

**Comment by tischi** [comment]
I see, if there is a reason, then fair enough :-)

**Comment by NicoKiaru** [comment]
I agree it does not look nice, but yeah it's annoying to modify now.
